index.html

```diff
@@ -29,8 +28,11 @@ <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
           <li><a href="#" class="icons"><img src="/images/assets/github.png"></a></li>
           <li><a href="#" class="icons"><img src="/images/assets/hi 5.png"></a></li>
         </ul><br>
+        <button type="button">Get my resume</button> 
```

Comment:
- Kindly remove this button as it's not specified in the [Figma design](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62) you should try to stay consistent with the design as much as possible. ✔